pkg/options/options.go

```diff
@@ -77,7 +54,7 @@ func GetOptionNames(opt interface{}) string {
 	return strings.Join(names, ",")
 }
 
-func newOptions(stringOptions string, opt interface{}) interface{} {
+func newOptions(stringOptions string, value bool, opt interface{}) interface{} {
```

Comment:
Can you rename the local variable `value` to `drain/clean` maybe (since that is what `true` indicates)?

Reply:
What do you think of something like `enabled`?

Reply:
I like that

Reply:
@devonboyer I just pushed the changes